tests/cluster_tests/AdaptiveSobol/test_adapt_sobol_parallel.xml

```diff
@@ -22,9 +22,7 @@
     </MultiRun>
     <IOStep name="print">
       <Input class="DataObjects" type="PointSet">solns</Input>
-      <Input class="Models" type="ROM">rom</Input>
       <Output class="OutStreams" type="Print">par_dump</Output>
-      <Output class="OutStreams" type="Print">par_dumprom</Output>
```

Comment:
Any reason here why rom output is not printed?

Reply:
I removed it because in the ```tests``` is not used

Reply:
(the only output that is checked for is ```output = 'workdir/par_dump.csv'```